src/main/java/mekanism/common/content/gear/ModuleConfigItem.java

```diff
@@ -52,10 +53,8 @@ public void set(@Nonnull TYPE val, @Nullable Runnable callback) {
             boolean checkModeState;
             if (name.equals(Module.ENABLED_KEY) && val == Boolean.TRUE) {
                 // disable other exclusive modules
-                if (module.getData().isExclusive()) {
-                    if (m.getData().isExclusive() && m.getData() != module.getData()) {
-                        m.setDisabledForce(callback != null);
-                    }
+                if (m.getData().isExclusive(module.getData().getExclusiveFlags()) && m.getData() != module.getData()) {
```

Comment:
I know it wasn't that way before but I feel like we might as well flip the order of this if statement and check `m.getData() != module.getData()` first before checking if they are exclusive of each other.